### PR TITLE
Add meta data to pages with placeholder text

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/RmOIhQqD/326-vv-ee-as-a-user-i-need-the-meta-information-to-reflect-the-site-content-so-that-i-can-search-for-or-share-the-site-easily-and-ac

## Description

- Add dynamic meta description and title
- Add Lang attribute
- Add og image (needs better url handling)

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Text.elm [hunk 1]
```diff
@@ -15,6 +15,86 @@ t key =
         SiteTitle ->
             "Economic Abuse Support Tool"
 
+        --- Site Meta
+        NotAlonePageMetaTitle ->
+            t SiteTitle
```
Comment (Nikomus) [inline src/Copy/Text.elm:20]: Neat

**Review by Nikomus — DISMISSED** [review]
Looks good to me 📦

**Comment by katjam** [comment]
I fixed a merge conflict. @Nikomus re-review...

**Review by Nikomus — APPROVED** [review]
Still looks good 👍

**Pull request merged**